src/packing.cpp

```diff
@@ -41,15 +41,15 @@ LowLevelRobotCommand createLowLevelRobotCommand(const proto::RobotCommand& comma
     llrc.do_kick = command.kicker();                                            // [0, 1]          {true, false}
     llrc.do_chip = command.chipper();                                           // [0, 1]          {true, false}
     llrc.kick_chip_forced = command.chip_kick_forced();                         // [0, 1] {true, false}
-    llrc.kick_chip_power = static_cast<int>(floor(kick_chip_power * 255 / 8));  // [0, 255]        [0, 100]%
+    llrc.kick_chip_power = static_cast<int>(floor(kick_chip_power * 255 / 6.5));  // [0, 255]        [0, 100]%
     llrc.velocity_dribbler = command.dribbler();                                // [0, 31]        [0, 100]%
 
     llrc.geneva_drive_state = command.geneva_state();  // [(0)1, 5]       [-2, 2]
     llrc.cam_position_x = 0;                           // [-4096, 4095]   [-10.24, 10.23]
     llrc.cam_position_y = 0;                           // [-4096, 4095]   [-10.24, 10.23]
     llrc.cam_rotation = 0;                             // [-1024, 1023]   [-pi, pi>
 
-    std::shared_ptr<proto::WorldRobot> findBot = utils::getWorldBot(command.id(), true, worldOpt);
+    std::shared_ptr<proto::WorldRobot> findBot = utils::getWorldBot(command.id(), isYellow, worldOpt);
```

Comment:
Please place a todo/some comment here, this seems outdated and is probably problematic anyways.

Reply:
I did not do this, maybe im on an old branch or smth?